tests/cupy_tests/creation_tests/test_from_data.py

```diff
@@ -500,6 +500,14 @@ def test_with_zero_size_array(self, dtype):
         assert a.data.ptr == 0
         assert a.size == 0
 
+    @testing.for_all_dtypes()
+    def test_asnumpy(self, dtype):
+        a = testing.shaped_arange((2, 3, 4), cupy, dtype)
+        b = DummyObjectWithCudaArrayInterface(a, self.ver, self.strides))
```

Comment:
I guess this syntax error made flake8 nuts?
```suggestion
        b = DummyObjectWithCudaArrayInterface(a, self.ver, self.strides)
```

Reply:
Thanks Leo! 😂